### PR TITLE
Oleksijt task3 calculate squares of numbers

### PR DESCRIPTION
**Pull request by Oleksijt**
Please check my script.

### CODE REVIEW

File: oleksijt/task3_calculate_squares_of_numers.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+def square_large_list(list_range):  # creates generator with max_range
+    for num in range(0, list_range):
+        yield num * num  # returns squared iterator
+
+
+max_range = 100000000000
+if sys.version_info < (3, 0, 0):
+    max_range = 1000000  # for Python 2.X max_range limited to 1000000
```
Comment (pivancy) [inline oleksijt/task3_calculate_squares_of_numers.py]: What do you mean under `for Python 2.X max_range limited to 1000000`?

Your code should work good enough in both Python versions (2.X,3.X) with exactly the same number - 100000000000

File: oleksijt/task3_calculate_squares_of_numers.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+ In result script runs in Python 3.6 an also Python 2.7.15.
+ Python 2.7.15 runs with max_range up to 1000000.
+ """
+
+import sys
+
+
+def square_large_list(list_range):  # creates generator with max_range
+    for num in range(0, list_range):
```
Comment (pivancy) [inline oleksijt/task3_calculate_squares_of_numers.py]: This doesn't make any sense `range(0, N`). It works in this way by default.
That's why it's much better to write just `range(N)`.

Comment (Oleksijt) [reply oleksijt/task3_calculate_squares_of_numers.py]: Script changed

File: oleksijt/task3_calculate_squares_of_numers.py (hunk 1 above)
Comment (Oleksijt) [reply oleksijt/task3_calculate_squares_of_numers.py]: Done. Now it works with 100000000000 in 2.7.15.
Is it right way?

**Comment by pivancy** [comment]
Interesting solution.

It might be replaced with `()` and `range, xrange` (see changes from others), but I also like your idea.

**Comment by pivancy** [comment]
👍

**Comment by Oleksijt** [comment]
> Interesting solution.
> 
> It might be replaced with `()` and `range, xrange` (see changes from others), but I also like your idea.

I saw other's code. But xrange works only with 10000000000 (10 zeros), but not with 100000000000 (11 zeros).

At least in my home computer such code brings OverflowError: Python int too large to convert to C long

def square_large_list(list_range):  # creates generator with max_range
    for num in xrange (list_range):
        yield num * num  # returns squared iterator


max_range = 10000000000
for number in square_large_list(max_range):
    print(number)

**Pull request merged**